Payments/UserInterface/05_CancellationsChargebacks.md

```diff
@@ -0,0 +1,7 @@
+# Cancellations and chargebacks
+
+
+
+The *Cancellations and chargebacks* menu entry is used to monitor refunds that have been initiated by the customer.   
+
+It only contains the [LIST](./04a_ListCancellationsChargebacks.md) tab.
```

Comment:
```suggestion
It only contains the [LIST](./05a_ListCancellationsChargebacks.md) tab.
```

Reply:
Aufwand für Dateiumbenennnungen steht in keinem Verhältnis zum Nutzen

Reply:
Done